feat(run_php): allow to set memory limit when using run_php (#621)
jolicode/castor: doc/going-further/helpers/run-php.md

php code of Castor.

The `run_php()` takes exactly the same options as the `run()` function.

### Script requiring more memory

> Maybe you need to execute a script that requires more memory than the default
> provided by PHP or castor static binary. In this case, you can add the
> `CASTOR_MEMORY_LIMIT` environment variable to increase the memory limit within
> the context.

Reviewer: ```suggestion
If you need to execute a script that requires more memory than the default
provided by PHP or castor static binary. In this case, add the
`CASTOR_MEMORY_LIMIT` environment variable to increase the memory limit within
the context:
```

Reviewer 2: The sentence is wrong in @lyrixx's patch :smile:

Author: Both fixed xD
